app/models/manageiq/providers/openshift/container_manager/container_template.rb

```diff
@@ -3,6 +3,9 @@
 class ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate < ManageIQ::Providers::ContainerManager::ContainerTemplate
   include ManageIQ::Providers::Kubernetes::ContainerManager::EntitiesMapping
 
+  # TODO: are these overridden?
+  # TODO: can we just focus on the unsupported_reason?
+  # TODO: can we just use supports_not :instantiate instead of using these methods?
```

Comment:
@kbrock I think we should keep this method for now since we override it in some scenarios but we could drop it in the future
Can you drop the TODO comments?